Explorer/Assets/DCL/Roads/Settings/Matrix4X4Comparer.cs

```diff
@@ -0,0 +1,45 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+
+namespace DCL.Roads.Settings
+{
+    public static class Matrix4X4Comparer
+    {
+        private const float EPSILON = 0.0001f;
+
+        public static readonly IEqualityComparer<Matrix4x4> DEFAULT = new EqualityComparer();
```

Comment:
Use the actual type instead of an interface to help the compiler avoid virtual calls.

Reply:
not needed. I removed it